images/linux/scripts/installers/basic.sh

```diff
@@ -12,4 +12,4 @@ for package in $common_packages $cmd_packages; do
     apt-get install -y --no-install-recommends $package
 done
 
-invoke_tests "Apt"
\ No newline at end of file
+invoke_tests "BasicTools"
```

Comment:
Why do we change linux scripts?

Reply:
Thank you, very good point. I've fixed it.